test/extended/crc/cmd/cmd_unix.go

```diff
@@ -0,0 +1,16 @@
+// +build !windows
+
+package cmd
+
+import "fmt"
+
+func (c Command) ToString() string {
```

Comment:
I played a bit with the code. Here is what I got:

```

func (c Command) ToString() string {
	cmd := append(c.env(), "crc", c.Action)
	return strings.Join(cmd, " ")
}

func (c Command) env() []string {
	var env []string
	if c.DisableUpdateCheck {
		env = append(env, envVariable("CRC_DISABLE_UPDATE_CHECK", "true"))
	}
	if c.DisableNTP {
		env = append(env, envVariable("CRC_DEBUG_ENABLE_STOP_NTP", "true"))
	}
	return env
}

func envVariable(key, value string) string {
	if runtime.GOOS == "windows" {
		return fmt.Sprintf("$env:%s=%s;", key, value)
	}
	return fmt.Sprintf("%s=%s", key, value)
}

func TestCommand(t *testing.T) {
	assert.Equal(t, "crc setup", Command{
		Action: "setup",
	}.ToString())
	assert.Equal(t, "CRC_DEBUG_ENABLE_STOP_NTP=true crc start -b bundle", Command{
		Action:     "start -b bundle",
		DisableNTP: true,
	}.ToString())
	assert.Equal(t, "CRC_DISABLE_UPDATE_CHECK=true CRC_DEBUG_ENABLE_STOP_NTP=true crc start -b bundle", Command{
		Action:             "start -b bundle",
		DisableNTP:         true,
		DisableUpdateCheck: true,
	}.ToString())
}

```

It avoids to have a different file for Windows and share a bit more the code.

Reply:
Cool seems fine, will add it.